demos-europe/demosplan-core
# bug(refs T36069): Fix attribute name for attachment type

**Ticket:** https://yaits.demos-deutschland.de/T36069

Type  is a reserved keyword for the name of the resource type and must not be used as attribute. The new attribute name is attachmentType (see https://yaits.demos-deutschland.de/T35484 for more information). 

### How to review/test
Export zip from regio and import it in EWM with the new Beteiligungsimport. The source statement should be linked in the statement list.
